### ISSUE TITLE
How low should the loss be that clear local conditioned wav can be generated?

### ISSUE THREAD

**Issue by DabiaoMa** [ibab/tensorflow-wavenet#312]
below 1.0 or very close to 0?

**Comment by DabiaoMa** [comment]
I used MFCC-like local condition with dimension 63 and sampling frequency 200hz, the local condition is reverse-convolutioned to the bias term. I also used the logistic dist mentioned in the the distilled wavenet paper.
By using logistic dist, the loss drops faster. After about 27000 steps, the training  loss is about 0.7, the validation loss is about 0.75.
But I can not generate expected wavs with test local conditions. 
I kind of think this is due to the reason that the error are accumulated during synthesis. Does anyone who have successfully implemented local condition have any idea what may be the cause of the failure?
Maybe 0.75 is not low enough?
